src/remotion/ParticleExplosion/ParticleExplosionTemplate.tsx

```
@@ -0,0 +1,299 @@
+                );
+
+            case 'shatter':
+                if (explosionTime < 0) {
+                    return (
+                        <h1 style={baseStyles}>
+                            {text}
+                        </h1>
+                    );
+                }
+                
+                const characters = text.split('');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)